core/src/test/java/org/elasticsearch/index/analysis/AnalysisModuleTests.java

```diff
@@ -275,4 +281,46 @@ public void testUnderscoreInAnalyzerNameAlias() {
             assertThat(e.getCause().getMessage(), equalTo("analyzer name must not start with '_'. got \"_invalid_name\""));
         }
     }
+
+    @Test
```

Comment:
We've been slowly removing `@Test` on the methods in favor of the method names starting with `test` so no need to add it here. It'd be normal to remove `@Test` from the other tests in this class as well.